lib/clerk/authenticate_request.rb

```diff
@@ -103,7 +103,7 @@ def resolve_cookie_token(env)
         token = verify_token(auth_context.session_token_in_cookie)
         return signed_out unless token
 
-        if token['iat'] < auth_context.client_uat
+        if token['iat'] < auth_context.client_uat.to_i
```

Comment:
:question: Is there a chance that `token['iat']` is `nil`? Comparing with nil will raise a `NoMethodError`.